backend/tracim_backend/views/agenda_api/radicale_proxy_controller.py

```diff
@@ -20,7 +20,11 @@
     RadicaleWorkspaceSubItemPathSchema
 from tracim_backend.views.core_api.schemas import UserIdPathSchema
 from tracim_backend.views.core_api.schemas import WorkspaceIdPathSchema
-
+from requests.auth import HTTPBasicAuth
+# dumb auth parameter, just to avoid radicale issue with empty
+# auth header
+RADICALE_HTTP_AUTH_USERNAME = 'tracim'
+RADICALE_HTTP_AUTH_PASSWORD = 'tracim'
```

Comment:
use `tracimpass` instead of `tracim` as a password: this make the code easier to understand

Reply:
done.